libFMS.F90

```diff
@@ -65,215 +72,458 @@ module fms
+                               fms_ensemble_manager_get_ensemble_pelist          => get_ensemble_pelist, &
+                               fms_ensemble_manager_ensemble_pelist_setup        => ensemble_pelist_setup, &
+                               fms_ensemble_manager_get_ensemble_filter_pelist   => get_ensemble_filter_pelist
+  use atmos_ocean_fluxes_mod, only: fms_atmos_ocean_fluxes_init => atmos_ocean_fluxes_init, &
+                               fms_atmos_ocean_type_fluxes_init => atmos_ocean_type_fluxes_init, &
+                               fms_atmos_ocean_fluxes_set_coupler_flux         => aof_set_coupler_flux
 
   !> data_override
-  use data_override_mod, only: data_override_init, data_override, &
-                               data_override_unset_domains, data_override_UG
+  use data_override_mod, only: fms_data_override_init          => data_override_init, &
+                               fms_data_override               => data_override, &
+                               fms_data_override_unset_domains => data_override_unset_domains, &
+                               fms_data_override_UG            => data_override_UG
 
   !> diag_integral
-  use diag_integral_mod, only: diag_integral_init, diag_integral_field_init, &
-                               sum_diag_integral_field, diag_integral_output, &
-                               diag_integral_end
+  use diag_integral_mod, only: fms_diag_integral_init       => diag_integral_init, &
+                               fms_diag_integral_field_init => diag_integral_field_init, &
+                               fms_sum_diag_integral_field  => sum_diag_integral_field, &
+                               fms_diag_integral_output     => diag_integral_output, &
+                               fms_diag_integral_end        => diag_integral_end
 
   !> diag_manager
   !! includes imports from submodules made public
-  use diag_manager_mod, only: diag_manager_init, send_data, send_tile_averaged_data, &
-                           diag_manager_end, register_diag_field, register_static_field, &
-                           diag_axis_init, get_base_time, get_base_date, need_data, &
-                           DIAG_ALL, DIAG_OCEAN, DIAG_OTHER, get_date_dif, DIAG_SECONDS,&
-                           DIAG_MINUTES, DIAG_HOURS, DIAG_DAYS, DIAG_MONTHS, DIAG_YEARS, &
-                           get_diag_global_att, set_diag_global_att, diag_field_add_attribute, &
-                           diag_field_add_cell_measures, get_diag_field_id, &
-                           diag_axis_add_attribute, diag_grid_init, diag_grid_end, &
-                           diag_manager_set_time_end, diag_send_complete, &
-                           diag_send_complete_instant, DIAG_FIELD_NOT_FOUND, &
-                           CMOR_MISSING_VALUE, null_axis_id
+  use diag_manager_mod, only: fms_diag_manager_init => diag_manager_init, &
```

Comment:
I think the format for the diag manager reoutines is `fms_diag` without the manager.  That's what we've been doing with the new routines on the dmUpdate branch.